test/parallel/test-assert.js

```diff
@@ -318,9 +318,9 @@ try {
 assert.equal(true, threw,
              'a.doesNotThrow is not catching type matching errors');
 
-assert.throws(function() {assert.ifError(new Error('test error'));});
-assert.doesNotThrow(function() {assert.ifError(null);});
-assert.doesNotThrow(function() {assert.ifError();});
+assert.throws(function() {assert.ifError(new Error('test error')); });
+assert.doesNotThrow(function() {assert.ifError(null); });
+assert.doesNotThrow(function() {assert.ifError(); });
```

Comment:
Should probably be `{ statement; }` if we are going to be doing this, but I'd prefer not to make these changes at all.